README.md

```diff
@@ -126,6 +126,15 @@ RSpec::OpenAPI.enable_example = false
 # Change `info.version`
 RSpec::OpenAPI.application_version = '1.0.0'
 
+# Set the info header details
+RSpec::OpenAPI.info_header = {
```

Comment:
How about just `.info`? Using the same name as the schema seems easier to understand/remember.

Reply:
Sure, I updated the last commit